src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSpec.scala

```diff
@@ -847,69 +858,8 @@ class ClusterMonitorSpec extends TestKit(ActorSystem("leonardotest")) with FlatS
       // starting a cluster should not touch IAM roles
       verify(iamDAO, never()).removeIamRolesForUser(any[GoogleProject], any[WorkbenchEmail], mockitoEq(Set("roles/dataproc.worker")))
       verify(iamDAO, never()).removeServiceAccountKey(any[GoogleProject], any[WorkbenchEmail], any[ServiceAccountKeyId])
-    }
-  }
-
-  // Pre:
```

Comment:
This test wasn't actually testing anything - since the cluster started in Starting status and ended in Starting status, the eventually immediately succeeded, meaning the clusterMonitorActor didn't do anything (so the JupyterDAO and isProxyAvailable was never interacted with)